kalite/coachreports/api_views.py

```diff
@@ -10,7 +10,7 @@
 from kalite.main.models import ExerciseLog, VideoLog, ContentLog, AttemptLog, UserLogSummary
 from kalite.facility.models import FacilityUser
 from kalite.shared.decorators.auth import require_admin
-from kalite.topic_tools import get_topic_leaves, get_exercise_cache, get_content_cache
+from kalite.topic_tools.content_models import get_topic_contents, get_topic_nodes
```

Comment:
True that... I always use manual search tools like `ack-grep --python` to find these things... trying to use the most broad match and then review results manually. Not sure if it's the best possible way, but it usually works.